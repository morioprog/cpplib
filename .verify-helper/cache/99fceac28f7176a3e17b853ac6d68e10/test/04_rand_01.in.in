11
2 6 9
1 8
2 4 5
0
0
1 7
1 10
1 1
0
1 3
1 2
100
1 9
1 4
0 5 82
0 1 60
0 1 69
1 6
1 6
1 0
1 1
1 1
1 8
0 3 80
1 10
1 1
1 1
0 1 11
1 0
1 8
0 5 22
0 9 100
0 3 40
0 2 68
0 7 28
1 10
1 8
1 7
0 3 82
1 1
1 0
1 5
0 4 23
0 8 43
1 5
1 2
1 10
0 7 20
1 6
0 6 5
0 1 74
1 9
0 6 43
0 6 94
0 8 68
1 3
0 5 27
0 7 3
1 7
1 3
1 9
1 10
1 7
1 9
1 9
0 3 4
0 10 49
1 7
0 3 69
0 8 47
1 3
0 7 4
0 8 98
0 3 87
0 8 97
0 7 58
0 10 61
1 7
1 0
1 5
1 0
1 9
1 6
0 10 40
1 6
0 3 82
0 2 60
0 2 35
0 9 16
0 1 30
0 3 86
0 6 36
0 3 39
0 7 90
0 7 93
0 3 45
0 2 83
1 7
0 6 33
0 8 61
0 4 62
0 1 91
0 9 68
0 7 91
1 5
0 1 69
0 4 13
1 4
1 8
1 1
1 3
0 9 91
