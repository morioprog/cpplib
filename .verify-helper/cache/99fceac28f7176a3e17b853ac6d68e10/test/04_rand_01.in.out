0
0
211
211
0
1102
1102
1102
422
1102
1102
0
1168
680
1600
1460
1600
0
1292
1533
1218
812
426
302
504
2819
504
302
1338
2819
302
302
2917
650
4559
0
3879
0
462
1154
1194
6274
6572
5649
9324
8910
1412
